apps/utr-backend/src/endpoints/entry/EntryRoute.ts

```diff
@@ -1,22 +1,27 @@
 import { Router } from "express";
-const router = Router();
 import { getUser, isAdmin } from "../utils/AuthenticationUtils";
 
-import logger = require("../../config/winston");
+import * as logger from "../../config/winston";
 
 import { Entry } from "@prisma/client";
 import { DistanceEntry } from "./EntryService";
 import ApplicationError from "../utils/ApplicationError";
 
-const EntryService = require("./EntryService");
-const EntryUtils = require("../utils/EntryUtils");
+import * as EntryService from "./EntryService";
+// import * as EntryUtils from "../utils/EntryUtils";
+
+export const router = Router();
 
 router.get("/search", function (req, res, next) {
+	console.log(req.query.jobname);
+	console.log(req.query.latitude);
+	console.log(req.query.longitude);
+	console.log(req.query.page);
```

Comment:
Wofür die Console.log's jetzt?

Reply:
Stimmt, hab ich wohl übersehen, können eigentlich raus. Waren zum debuggen um zu sehen, welcher type die jeweils sind in dem Moment. 
Danke.

Reply:
Done